concordium-node/README.md

```diff
@@ -134,19 +172,22 @@ However, running the node requires the consensus DLL, which is compiled to `..\c
 If you wish to run the node directly, you should copy `HSdll.dll` to the location of `concordium-node.exe`.
 The node also depends on a number of DLLs that are part of `stack`'s `msys2` installation.
 To run the node with these DLLs in the path, you can use `stack exec`, as in:
-```
+
+```bash
```

Comment:
It would be good to be consistent if adding these. Either use `bash` or `shell` or `console` consistently.

Reply:
agree, do you have any preference?

Reply:
Chosen `console`